labellab-client/src/components/labeller/index.js

```diff
@@ -115,7 +115,9 @@ class LabelingLoader extends Component {
             labels={lab}
             labelData={(img && img.labelData) || {}}
             imageUrl={
+              'http://' +
```

Comment:
It could be either http/https when it will be production stage. So, it's better to include that in HOST variable.

Reply:
@Tushar19varshney Sure, that makes sense. `FetchAPI.js` contains the following code:
```
if (process.env.REACT_APP_SERVER_ENVIORNMENT === 'dev') {
    url =
      'http://' +
      process.env.REACT_APP_HOST +
      ':' +
      process.env.REACT_APP_SERVER_PORT +
      url
  }
```
and `project/images.js` also adds `http://` to `process.env.REACT_APP_HOST`. Should I make the necessary changes to these files instead, so that we can keep `REACT_APP_HOST=http://localhost:` during development?

Reply:
Yes and also add the instructions in the `README.md` so that it will be easy for other developers to understand.